setup/includes/upgrades/common/3.0.0-remove-tv-eval-system-setting.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
My PR is not merged yet but after I would suggest using one common upgrading script for removing obsolete system settings that never used. Example here https://github.com/modxcms/revolution/pull/13859/files#diff-ce54080b4829628803023b4d30b21700

Reply:
No, it should not use one common upgrade script. Dependencies between PRs is not a good idea. If one needs to be reverted, then that makes the process more complicated.

Reply:
It will violate the DRY principles but make sense according to maintenance. But at least script in this PR should follow your recommendations that you left in my PR.

Reply:
Agreed.